### ISSUE TITLE
Viewport display modes not working in GLES 2

### ISSUE THREAD

**Issue by hjsrabi** [godotengine/godot#27018]
**Godot version:**

3.1

**OS/device including version:**

Pop OS 18.10/ NVIDIA GeForce 920MX

**Issue description:**

when you try to change the viewport render mode to something like wireframe with the gles 2 renderer nothing happens
![Screenshot from 2019-03-13 19-08-37](https://user-images.githubusercontent.com/47189897/54300196-079b4180-45c5-11e9-8ea7-30e0fe619a08.png)

when i tried it in gles 3 it worked fine
![Screenshot from 2019-03-13 19-24-26](https://user-images.githubusercontent.com/47189897/54300492-a758cf80-45c5-11e9-8416-af9aa3ef5a0d.png)

**Steps to reproduce:**
create a new project
switch to gles 2
add a mesh instance
change the viewport display mode to wireframe, overdraw, unshaded.

**Comment by clayjohn** [comment]
Yes. This is a known bug. It was reported here. https://github.com/godotengine/godot/issues/26406. Although you explain it much better. :)

**Comment by Toshiwoz** [comment]
I'm working on a project that relies on gles2.0 would be great to have it fixed.
Is wireframe not possible because of some limitations on this version of the graphic library?

**Comment by clayjohn** [comment]
@Toshiwoz It just hasn't been done yet! It is a lot of work.

**Comment by CarlGustavAlbertDwarfsteinYung** [comment]
I can confirm only Normal works. 

Note that Orthogonal doesn't work either in GLES2 editor viewport.  It just displays everything as clear color.

**Comment by KoBeWi** [comment]
Still valid in 3.2.4 beta3
No longer relevant for 4.0, as GLES2 for 3D is not planned.

**Comment by Calinou** [comment]
> No longer relevant for 4.0, as GLES2 for 3D is not planned.

To clarify, we don't know for certain whether GLES2 will support 3D in 4.0. See https://github.com/godotengine/godot-proposals/issues/877.
